File: Day9/FirstAPISolution/FirstAPiApp/Models/Employee.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace FirstAPiApp.Models
{
    public class Employee
    {
        public int Id { get; set; }
        public string Name { get; set; } = string.Empty;
        public string Email { get; set; } = string.Empty;
        public int? DepartmentId { get; set; }

        [ForeignKey("DepartmentId")]
        public Department? Department { get; set; }
        
    }
}

File: Angular/employeeapp/src/app/models/employee.ts
// export class Employee{
//    public id: number;
//    public name: string;
//    public  age: number;
//    constructor(id: number=0, name: string="", age: number=0){
//        this.id = id;
//        this.name = name;
//        this.age = age;
//    }
// }

export class Employee{

    constructor(public id: number=0, 
        public name: string="", 
        public age: number=0,
        public image:string="",
        public email:string=""){

    }
 }